src/pages/Swap/Components/ChartTabs.tsx

```diff
@@ -59,7 +57,7 @@ export const ChartTabs = ({
           navigate('/swap')
         }}
         title={t('advancedTradingView.chartTabs.offTitle')}
-        disabled={!hasBothCurrenciesInput}
+        disabled={proOptionsDisabled}
```

Comment:
Should we let OFF button enabled even in mobile? Having a mixed feeling about this

Reply:
I also have a mixed feeling, but I was thinking a mobile user might know atleast that a Pro mode is available if he goes on to desktop. 🤔 Should we add a tooltip upon tapping anywhere in this tab?

Reply:
I'm saying this because in mobile doesn't highlight the OFF button, both are disabled and seems odd. Maybe a tooltip saying not enabled on mobile.

Reply:
I added the tooltip and extented mouseover tooltip to include onClick as well since we would want users on mobile to have tooltips when they click